app/models/user.rb

```diff
@@ -2,4 +2,8 @@ class User < ApplicationRecord
   has_many :comments
   has_many :posts
   has_many :likes
```

Comment:
- [ ] Good job working on these associations, but please kindly include a reference to their foreign key.

Hint:
`has_many :posts, foreign_key: 'author_id'`

Reply:
Thank you for requesting this change